### DISCUSSION TITLE
Warning: Key LABEL_COLON not found in the i18n bundle, the default text will be used

### DISCUSSION THREAD

**Discussion by Roba1993** [UI5/webcomponents#7322] [Q&A]
I get quiet often the following warning:
<img width="1568" alt="Bildschirmfoto 2023-07-12 um 12 59 27" src="https://github.com/SAP/ui5-webcomponents/assets/1376958/2e45e424-18b2-415a-b6b5-c977c73b3e85">

Can you help me, on what I'm doing wrong and how I can fix that?

**Comment by irodrigob** [comment]
I have the same problem. I thought the problem was with the <Label for="..." component. But I remove it and still the same warning.

**Comment by pop1989bb** [comment]
I have the problem too. Would like to have a resultion as well. :)

**Comment by Lukas742** [comment]
This will be fixed with https://github.com/SAP/ui5-webcomponents/pull/7652

**Comment by 0anton** [comment]
Any timeline to release this? We are getting 1000 warning (really!) on a single page load, which very negatively affects performance of the app.

**Comment by Lukas742** [comment]
Hi all, this was fixed in [v1.19.0](https://github.com/SAP/ui5-webcomponents/releases/tag/v1.19.0), even though it doesn't show up in the release log for that version.

https://github.com/SAP/ui5-webcomponents/pull/7652#issuecomment-1820417630

**Comment by Roba1993** [comment]
Thank you @Lukas742

**Comment by draysams** [comment]
This issue is now happening in ~v2.7.0

![Screenshot 2025-04-22 at 16 12 49](https://github.com/user-attachments/assets/28b04673-14af-4706-beda-9e74530db55d)

**Reply by giuliamodugno** [reply]
Hi, is this solved? it makes the console useless....
